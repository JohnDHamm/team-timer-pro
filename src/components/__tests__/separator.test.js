import React from 'react'
import renderer from 'react-test-renderer'
import Separator from '../separator'

describe('<Separator />', () => {
  const width = 150;
  const color = 'green';

  test('it renders correctly', () => {
    const tree = renderer.create(<Separator width={width} color={color} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
