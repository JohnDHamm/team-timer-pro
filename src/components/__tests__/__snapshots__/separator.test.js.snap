// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> it renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "green",
      "borderColor": "transparent",
      "borderWidth": 1,
      "height": 1,
      "marginBottom": 10,
      "width": 150,
    }
  }
/>
`;
